src/material.js

```diff
@@ -460,6 +487,16 @@ class gltfMaterial extends GltfObject
             this.clearcoatNormalTexture = clearcoatNormalTexture;
         }
     }
+
+    fromJsonMetallicRoughnessSpecular(jsonMRSpecular)
```

Comment:
why is this method (and parameter) called `fromJsonMetallicRoughnessSpecular` when the extension is just called `specular`?